managers.py

```diff
@@ -0,0 +1,43 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema")
+        self.table_name = "actors"
+
+    def create(self, first_name_: str, last_name_: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name_, last_name_)
+        )
+        self._connection.commit()
+
+    def all(self) -> list:
```

Comment:
Specify type for `list`
```suggestion
    def all(self) -> list[Actor]:
```